src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/PackAllRids.targets

```diff
@@ -4,6 +4,23 @@
     <ImportNuGetBuildTasksPackTargetsFromSdk>false</ImportNuGetBuildTasksPackTargetsFromSdk>
     <_RoslynPublishReadyToRun>false</_RoslynPublishReadyToRun>
     <_RoslynPublishReadyToRun Condition="'$(Configuration)' == 'Release'">true</_RoslynPublishReadyToRun>
+
+    <!-- 
+      List of runtime identifiers that we want to publish an executable for.
+      This cannot be set in the base project as it will cause issues when roslyn is built
+      as a test project for SDK insertions.  The RuntimeIdentifiers property will cause the build to
+      attempt to download unpublished packages, which fails (until the packages eventually get published).
+
+      The test doesn't actually run pack, so we can instead skip setting the RuntimeIdentifiers property unless
+      we're actually trying to build packages.
```

Comment:
Does this change still break our release builds though, since I'd expect the runtime packs get used in the ready-to-run publish? but maybe not?

Reply:
I was able to do a restore/build/pack in release, and the ci is green. Is there something else that needs validating?

Reply:
That'd all I can think of to test, so probably good.